utils/index.ts

```
@@ -98,15 +104,15 @@ const apiKey = model === 'grok-2-latest' ? process.env.X_AI_API_KEY : model ===
     const statusText = res.statusText;
     const result = await res.body?.getReader().read();
     const errorMessage = decoder.decode(result?.value) || statusText;
-    if (model === 'grok-2-latest') {
+    if (model === 'grok-2-latest' || 'grok-3-mini-beta') {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)